### PR TITLE
[Do Not Merge] release-1.6

### PR DESCRIPTION
**Pull request by caesarxuchao**
Generated by bot.

### CODE REVIEW

**Comment by deads2k** [comment]
Missing a kubernetes-sha update.

**Comment by caesarxuchao** [comment]
The kubernetesSha isn't updated because the bot doesn't find any commit to cherry-pick.

If we are to record the level of kubernetes the repo is synced to, we'll record it in another meta file.

**Comment by deads2k** [comment]
> The kubernetesSha isn't updated because the bot doesn't find any commit to cherry-pick.
> 
> If we are to record the level of kubernetes the repo is synced to, we'll record it in another meta file.

You're changing the levels of vendored code though.  With this update, it's no longer the case that kubernetes-sha listed matches the code contained.  This repo now contains code that is *not* in the kubernetes-sha that is provided.

**Comment by caesarxuchao** [comment]
From the robot's perspective, kubernetes-sha doesn't need (and shouldn't) to be updated, because the purpose of this meta file is to help find where to start cherry-picking in the next run of the robot.

From the user's perspective, if he wants to find the matching kubernetes level, currently the only way is finding the latest sha among all kubernetes-shas in this repo and the k8s.io repos it depends on.

In short, we need another meta file to help human finding the matching kubernetes level. We can do that in another PR.

**Comment by deads2k** [comment]
I'm not really convinced, but I'm not going to block turning on the auto-syncing for it.

You need these merged manually or will the bot get it?

**Comment by caesarxuchao** [comment]
No, it's just for sanity check.

Let me try to explain it in another way, kubernetes-sha is the last commit that affects /staging/.../kube-aggregator, the bot doesn't find any commit to cherry-pick, so kubernetes-sha is not updated.

Yes, the vendor/ is updated, but staging/.../kube-aggregator/vendor/ doesn't include these changes, so there is no commit to be cherry-picked.
